src/main/resources/application-oauth.properties

```diff
@@ -6,11 +6,11 @@ logging.level.org.springframework.web=info
 logging.level.org.springframework.security=debug
 # set authorization-token-type: JWT or OpaqueToken
 authorization.token.type=JWT
-# set authorization server endpoint and client configs+
-authorization.server.endpoint=
-authorization.api.client.id=
-authorization.api.client.secret=
-authorization.swagger.ui.client.id=
+# set authorization server endpoint and client configs
+authorization.server.endpoint=https://iam.xpanse.site
```

Comment:
we must remove all these defaults. It was purposely kept empty to ensure that terraform-boot is an independent project.

Reply:
Restored.